packages/toolkit/src/createAsyncThunk.ts

```diff
@@ -744,3 +772,31 @@ function isThenable(value: any): value is PromiseLike<any> {
     typeof value.then === 'function'
   )
 }
+
+/**
+ * An error serializer function that can be used to serialize errors into plain objects.
+ *
+ * @param error - The error to serialize
+ * @param defaultSerializer - The original default serializer `miniSerializeError` https://redux-toolkit.js.org/api/other-exports/#miniserializeerror
+ *
+ * @public
+ */
+type ErrorSerializer<ThunkApiConfig extends AsyncThunkConfig> = (
+  error: any,
+  defaultSerializer: (error: any) => SerializedError,
+) => GetSerializedErrorType<ThunkApiConfig>
+
+/**
+ * @public
+ */
+type CreateAsyncThunkCreatorOptions<ThunkApiConfig extends AsyncThunkConfig> = {
+  serializeError?: ErrorSerializer<ThunkApiConfig>
+}
+
+export const createAsyncThunkCreator = /* @__PURE__ */ (() => {
```

Comment:
why have this instead of just exporting (and renaming) internalCreateAsyncThunkCreator?